tensorflow/lite/toco/tflite/operator.cc

```diff
@@ -953,7 +953,7 @@ class Lstm : public BuiltinOperator<LstmCellOperator, ::tflite::LSTMOptions,
   flatbuffers::Offset<TfLiteOptions> WriteOptions(
       const TocoOperator& op,
       flatbuffers::FlatBufferBuilder* builder) const override {
-    ::tflite::LSTMKernelType kernel_type;
+    ::tflite::LSTMKernelType kernel_type = ::tflite::LSTMKernelType_FULL;
```

Comment:
It has been updated in current code. Please update your pr.

Reply:
@lu-wang-g , thanks for the review the PR is quiet old, so it needed to be re-based, i have done the same and removed the already existing code.

Regards
Amit